android/app/src/main/java/com/bounswe2020group3/paperlayer/login/LoginFragment.kt

```diff
@@ -6,20 +6,65 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.Button
-import androidx.navigation.Navigation
+import android.widget.EditText
 import com.bounswe2020group3.paperlayer.R
 
-class LoginFragment : Fragment() {
+class LoginFragment : Fragment(),LoginContract.View {
+
+    //My presenter object
+    private lateinit var mPresenter: LoginPresenter
+    //My view
+    private lateinit var mView: View
+
+    //My UI elements
+    private lateinit var loginButton: Button
```

Comment:
- You dont need to define variables for each element in the UI, you can use their id and it wil automatically import using synthetic extensions